_posts/2017-07-21-pydelhi-conf-2017-a-beautiful-conference-happend-at-new-delhi-india.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: post
+title: "Pydelhi Conf 2017: A beautiful conference happened in New Delhi, India"
+date: "2017-07-21 18:14:03 +0530"
+tag:
+  - python
+  - conference
+  - django
+  - talks
+---
+
+![Pydelhi Conf
+2017]({{site.url}}/assets/images/pydelhi_conf_2017/group_photo.jpg)
+
+## TL;DR
+
+[PyDelhi conf 2017][pydelhi_conf_2017] was a two-day conference which served
+talks, lightning talks, workshops and dev sprint. If you are a beginner,
+attending this conference is worth because there are free workshops. If you are
+an experienced, attending this conference is worth because of curated advanced
+level talks. Delhiite should not miss the chance to attend this conference. I
+conducted a workshop titled **“Tango with Django”** helping beginners to
+understand Django web framework.
```

Comment:
Try ` understand the Django web framework` or, `understand Django, a web framework`.